src/opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -819,6 +820,7 @@ SummaryConfigNode::Category parseKeywordCategory(const std::string& keyword) {
     if (is_special(keyword)) { return Cat::Miscellaneous; }
```

Comment:
None of the keywords checked in `is_special` currently start with `A`; is this because aquifer keywords have not been supported, or is this because there are no keywords in `SUMMARY` that can start with `A` other than aquifer keywords?

Reply:
> is this because there are no keywords in SUMMARY that can start with A other than aquifer keywords?

Until proven otherwise I would say it is like this - i.e. all legal SUMMARY keywords starting with 'A' are aquifer keywords.

Reply:
> is this because aquifer keywords have not been supported, or is this because there are no keywords in SUMMARY that can start with A other than aquifer keywords?

Both.  Flow does not create any kind of aquifer-related output&mdash;neither to the summary nor to the restart files.  We have (very) limited support for **reading** parameters of the Fetkovich analytic model **from** a restart file&mdash;I needed that in a different project&mdash;but writing is unsupported at present.  There is a lot of work ahead of us before we can reasonably claim to fully support aquifer workflows.

Writing restart data for aquifers is on my plate.

Reply:
Then I will at this time leave `is_special()` as is.